lib/util/device-service-details.ts

```diff
@@ -82,16 +86,21 @@ function getSingleInstallSummary(
 	) {
 		const release = (rawData.is_provided_by__release as Release[])[0];
 		releaseInfo = {
-			commit: release != null ? release.commit : void 0,
+			commit: release != null ? release.commit : undefined,
 		};
 	}
 
+	// prefer over omit for performance reasons
+	delete rawData.image;
+	if ('is_provided_by__release' in rawData) {
```

Comment:
As discussed in FD, this is so that TS is happy about `is_provided_by__release` possibly being `?` in the  `CurrentService | CurrentGatewayDownload` union.